.vscode/settings.json

```diff
@@ -8,6 +8,13 @@
   "eslint.enable": true,
   "eslint.workingDirectories": [{ "mode": "auto" }], // infer working directory based on .eslintrc/package.json location
 
+  "explorer.fileNesting.enabled": true,
+  "explorer.fileNesting.patterns": {
+    "*.js": "${capture}.js.map, ${capture}.d.ts, ${capture}.d.ts.map",
+    "*.ts": "${capture}.android.tsx, ${capture}.ios.tsx, ${capture}.macos.tsx, ${capture}.windows.tsx, ${capture}.win32.tsx, ${capture}.android.ts, ${capture}.ios.ts, ${capture}.macos.ts, ${capture}.windows.ts, ${capture}.win32.ts",
```

Comment:
Do we want to add mobile to this list?

Reply:
Also I think it makes sense to separate *.ts and *.tsx. I don't think a file makes sense to be forked and not have the same extension.

Reply:
Yeah, this is a proposal. The goal should be to make it easier for FURN/RN devs to understand RN structure. But I'm also perfectly happy to leave this as a personal setting that individual devs have to add themselves..

I think the reason ".ts" and ".tsx" are grouped together (according to the tweet) is because the files will get transpiled to JS, and be picked up by metro all the same. I haven't confirmed but it sounds like these 3 files like 
- AvatarTokens.js
- "AvatarTokens.ios.**tsx**
- AvatarTokens.android.**ts**
Speaking of, I omitted the `${capture}.<platform>.js` variants from this list but I could add them back.

Reply:
Also note, enabling file nesting means other side effects happen,  like "yarn.lock" being nested under "package.json"...

Reply:
Hm. Not sure how much I like that, but I guess it's just something you get used to.

Reply:
Adding Tommy and Coates for some thinking towards this. In general, I like the idea but would need to play more with how 'discoverable' the platform forks are for more junior folks.

Reply:
I saw the original tweet, thought it was an interesting idea.  Wont know for sure what I think of it until I've used it.  -- It's probably a fairly harmless experiment to try anyway.

Reply:
I have some concerns with discoverability as well, but it does look nice. Maybe we can enable `explorer.fileNesting.expand` as well?

Reply:
My other concern is that file nesting looks too much like a directory, and that will confuse people into thinking there's another level in the folder hierarchy. But I guess that was always a problem with this setting ¯⁠\⁠_⁠(⁠ツ⁠)⁠_⁠/⁠¯